src/main/paradox/features/akka-cluster-bootstrap.md

```diff
@@ -0,0 +1,44 @@
+## Akka Cluster Bootstrap
+
+When enabled, this feature will use service discovery to identify other contact points. Once the requisite number of
```

Comment:
Explain that this enables akka-cluster-bootstrap which is documented in depth here: https://developer.lightbend.com/docs/akka-management/current/bootstrap.html